docs/apis/addin-management-apis.md

```diff
@@ -0,0 +1,325 @@
+| addinPermissions | SPAddinPermissionInfo[]       | The returned permissions.                                             |
+| failedAddins     | SPAddinPermissionFailedInfo[] | The Add-ins that failed to get permissions and corresponding reasons. |
+
+#### SPAddinPermissionInfo
+
+| Name                            | Type                                   | Description                                                                                                                                                                                        |
+|---------------------------------|----------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| tenantScopedPermissions         | SPTenantScopedPermissionInfo[]         | This is the permissions grant in tenant scope level.                                                                                                                                               |
+| siteCollectionScopedPermissions | SPSiteCollectionScopedPermissionInfo[] | This is the permissions grant in site collection scope level.                                                                                                                                      |
+| appIdentifier                   | string                                 | The identifier of the Add-in.                                                                                                                                                                      |
+| serverRelativeUrl               | string                                 | The server relative url of the site collection.                                                                                                                                                    |
+| allowAppOnly                    | bool                                   | This identifies if the Add-in allows app only mode. For more information, see [Add-in authorization policy types in SharePoint](../sp-add-ins/add-in-authorization-policy-types-in-sharepoint.md). |
+
+#### SPTenantScopedPermissionInfo
+
+| Name    | Type   | Description                                                                                                                                                                                         |
+|---------|--------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| feature | string | The feature name of the permissions, it should be one of these:<br/> <li> Taxonomy <br/> <li> Social <br/> <li> ProjectServer <br/> <li> Search <br/> <li> BcsConnection <br/> <li> Content         |
```

Comment:
This isn't a valid bullet list. Remove the `<br />` tags and surround all `<li>` with the correct `<ul>` element.

Reply:
Updated, thanks!